examples/chip-tool/commands/clusters/ClusterCommand.h

```diff
@@ -64,12 +65,30 @@ class ClusterCommand : public ModelCommand, public chip::app::CommandSender::Cal
 
     CHIP_ERROR SendCommand(ChipDevice * device, std::vector<chip::EndpointId> endpointIds) override
     {
-        return ClusterCommand::SendCommand(device, endpointIds.at(0), mClusterId, mCommandId, mPayload);
+        return InteractionModelCommands::SendCommand(device, endpointIds.at(0), mClusterId, mCommandId, mPayload,
+                                                     mTimedInteractionTimeoutMs, mRepeatCount, mRepeatDelayInMs, mSuppressResponse);
+    }
+
+    template <class T>
+    CHIP_ERROR SendCommand(ChipDevice * device, chip::EndpointId endpointId, chip::ClusterId clusterId, chip::CommandId commandId,
+                           const T & value)
+    {
+        return InteractionModelCommands::SendCommand(device, endpointId, clusterId, commandId, value, mTimedInteractionTimeoutMs,
+                                                     mRepeatCount, mRepeatDelayInMs, mSuppressResponse);
     }
 
     CHIP_ERROR SendGroupCommand(chip::GroupId groupId, chip::FabricIndex fabricIndex) override
     {
-        return ClusterCommand::SendGroupCommand(groupId, fabricIndex, mClusterId, mCommandId, mPayload);
+        return InteractionModelCommands::SendGroupCommand(groupId, fabricIndex, mClusterId, mCommandId, mPayload,
+                                                          mTimedInteractionTimeoutMs);
```

Comment:
This doesn't make sense, really: group commands are never supposed to be timed interactions.  Is this there just to test the invalid case?

Reply:
No that was a mistake. I have updated all the "group" methods to not contains the timed interaction parameter.